### DISCUSSION TITLE
希望頂欄添加wiki

### DISCUSSION THREAD

**Discussion by ZOMBIE39** [the1812/Bilibili-Evolved#1820] [功能建议 / Ideas]
著迷wiki沒了之後 很多手游的wiki組都轉到B站wiki了 但wiki要進去挺麻煩的 一直不知道具體的鏈接在哪 所以希望頂欄上添加wiki

**Comment by Coulomb-G** [comment]
https://wiki.biligame.com
是指这个吗?

**Reply by ZOMBIE39** [reply]
> https://wiki.biligame.com
> 是指这个吗?

對 就是這個 也不知道B站把這個鏈接放哪裡的 每次想進去 都得百度搜一下

**Comment by the1812** [comment]
浏览器可以添加到收藏夹
![image](https://user-images.githubusercontent.com/26504152/117411249-f6041900-af45-11eb-904e-f9bbb45eedb1.png)

b站都不需要进, 网址直接输入`game`或`wiki`就会跳出补全的

**Reply by ZOMBIE39** [reply]
> 浏览器可以添加到收藏夹
> ![image](https://user-images.githubusercontent.com/26504152/117411249-f6041900-af45-11eb-904e-f9bbb45eedb1.png)
> 
> b站都不需要进, 网址直接输入`game`或`wiki`就会跳出补全的

收藏夾那一排太滿了 我把B站設置成首頁了 如果能有個wiki入口的話 會方便很多
